### DISCUSSION TITLE
Internship or Summer courses for Students joining in Spring

### DISCUSSION THREAD

**Discussion by AditiJain111** [snigi-gupta/AttendingUniversityAtBuffalo#63] [Q&A]
Are students eligible to do an internship in Summer 2022 if they join the MS in CSE program in Spring 2022 (intending to complete the course in 1.5 years)? If not, are they allowed to take up any courses in Summer?

Also, can you please tell how much was the total cost of 1.5 yrs (tuition + living) for MS in CS ?

**Comment by snigi-gupta** [comment]
@AditiJain111 
Are students eligible to do an internship in Summer 2022 if they join the MS in CSE program in Spring 2022 (intending to complete the course in 1.5 years)?
Ans: No, to work on CPT you need to be in the country for at least 1 yr, so spring students cannot do an internship

If not, are they allowed to take up any courses in Summer?
Ans: Yes, you can take course during the summer to lighten your course load during the last sem

Also, can you please tell how much was the total cost of 1.5 yrs (tuition + living) for MS in CS ?
Ans: $42K

Apologies for the late response.
